### ISSUE TITLE
Remove start_pes section

### ISSUE THREAD

**Issue by jamesaross** [openshmem-org/specification#404]
The `start_pes` routine is deprecated since 1.2, but not removed.  Using the same logic to remove the cache management section, should remove the `start_pes` section?  The document doesn't have sections on related deprecated routines `_my_pe`, `_num_pes`.  Rather, they have notes in their replacement routines `shmem_my_pe` and `shmem_n_pes`, respectively.  Similarly, there is already a note in the `shmem_init` section about `start_pes` being deprecated.

**Comment by naveen-rn** [comment]
Related item: I think, as a community we should start deciding on the versioning schemes. We don't have backward compatibility on the planned release with the existing 1.4 release. We have removed support for FTN, cache, and then this request to remove ```start_pes```. We release 1.5 as an update to 1.4, but in fact we have removed compatibility.

**Comment by jamesaross** [comment]
@naveen-rn, Hasn't Fortran been the only thing removed from 1.5?  I think it has been the only thing removed since 1.0. The cache management routines and `start_pes` are deprecated, not removed.  I only proposed to remove the `start_pes` _section text_ within the specification, not remove it from backward compatibility support.  If you look at the `shmem_init` section, it details how `start_pes` has been deprecated.  It's been deprecated for a while and as @nspark mentions in #378, we've purged other deprecated API sections from the specification document.  But software support remains.

**Comment by naveen-rn** [comment]
@jamesaross are you saying that implementations might still need to support cache management routines in OpenSHMEM-1.5? I'm confused with how it is represented in https://github.com/BryantLam/openshmem-specification/pull/9/files

**Comment by jamesaross** [comment]
I missed that.  Obviously, a little clarity is needed.  I don't recall voting on removing the cache API.  I didn't know removal could be done by a section committee.  I thought #378 was for removing the deprecated section text.

**Comment by jdinan** [comment]
@jamesaross I agree -- in general, removal should not be something we handle through section committee. At the face-to-face, we decided to make an exception and garbage collect the cache APIs since they have not been used in 20 years. Looking at @BryantLam's back matter branch, it looks like additional APIs are being removed in 1.5. I suggest we discuss this in today's WG session. Can you join?

**Comment by jamesaross** [comment]
Personally, I have no concerns about removing the cache management API.

Somehow, I don't recall that discussion at the F2F meeting. It wasn't included in 1.5rc1 and it wasn't mentioned at the March 20 committee meeting.  Unless you interpret the pull request #378 discussion correctly or parse unmerged LaTeX diffs on an external fork where a single line notes removal, it's like backward compatibility for the C API was broken without most of the community being aware.  Let's make this very clear at the next meeting.
